utils/git.go

```diff
@@ -276,8 +278,6 @@ func (m *manager) readRevisionFromPackedRef(ref string) {
 			return
 		}
 	}
-
-	m.err = errorutils.CheckError(errors.New("failed fetching revision from git config, from ref: " + ref))
```

Comment:
I suggest adding a log message instead of this error. Something like "No packed-refs file was found. Assuming git repository is empty"